src/Utility/TraitsByClassHelperTrait.php

```diff
@@ -36,4 +38,16 @@ final protected function getTraitsByClass(string $class)
 
         return $this->_traitsByClasses[$class];
     }
+
+    final protected function checkClassTypeHasTrait(?Class_ $class, string $trait): bool
+    {
+        if ($class === null) {
+            return false;
+        }
+        $className = $this->nodeNameResolver->getName($class);
+        return in_array(
+            $trait,
+            $this->getTraitsByClass($className)
+        );
+    }
```

Comment:
Previously the code was just checking if the class name was an empty string. Now Rector leverages a type object lookup. This helps keep code clean when checking if a class node has a trait.